website/www/site/content/en/documentation/io/built-in/snowflake.md

```diff
@@ -126,10 +121,6 @@ Snowflake IO library supports following options that can be passed via the [comm
 
 `--password` Required for username/password authentication only.
 
-`--privateKeyPath` Path to Private Key file. Required for Private Key authentication only.
```

Comment:
I think `privateKeyPath` should stay, only `rawPrivateKey` should be removed.

Reply:
Corrrect :)

Reply:
re-added